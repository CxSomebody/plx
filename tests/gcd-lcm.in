509048
823657
