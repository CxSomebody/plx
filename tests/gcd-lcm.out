Please input a, b
gcd(a,b)=197
lcm(a,b):
2128329688
